plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -403,4 +407,15 @@ public static Transaction newCreateTableTransaction(TrinoCatalog catalog, Connec
 
         return catalog.newCreateTableTransaction(session, schemaTableName, schema, partitionSpec, targetPath, propertiesBuilder.buildOrThrow());
     }
+
+    public static void validateTableCanBeDropped(Table table, SchemaTableName schemaTableName)
```

Comment:
No need to pass `SchemaTableName` here, since we construct `Table` objects with the correct name.